### ISSUE TITLE
No definition for objects that are items in an array

### ISSUE THREAD

**Issue by atruskot** [Redocly/redoc#2040]
**Describe the bug**
Building a static HTML page with recent releases of redoc-cli prevents my spec from rendering the definitions of objects that are found in an array.

You can see a new `[ items <objectName> ]` label added after "Array of identifiers" description. Were new settings introduced to toggle some new behavior? 

**Expected behavior**
See first screenshot below. Expectation is to see the definition of the object. 

**Minimal reproducible OpenAPI snippet(if possible)**
```
openapi: 3.1.0
servers:
  - url: http://server.com
    description: the server
info:
  title: Issue
  version: 1.0.0
  description: min spec
paths:
  /newPath:
    post:
      operationId: opId
      summary: an operation
      requestBody:
        content:
          application/json:
            schema:
              properties:
                identifier:
                  type: array
                  minItems: 1
                  description: >
                    An identifier
                  items:
                    $ref: "#/components/schemas/Identifier"
components:
  schemas:
    Identifier:
      type: object
      title: Identifier
      format: Identifier
      properties:
        system:
          type: string
          description: a system
          pattern: >-
            [ \r\n\t\S]+
        value:
          type: string
          description: a value
          pattern: >-
            [ \r\n\t\S]+

```

**Screenshots**
`redoc-cli@0.13.3`
![image](https://user-images.githubusercontent.com/95640878/171757169-db2077a1-2bc6-49da-8d72-35c3b85a59e7.png)

`redoc-cli@0.13.15`
![image](https://user-images.githubusercontent.com/95640878/171757028-02332801-5793-4a80-a4bc-95f033e1abdf.png)

**Comment by AlexVarchuk** [comment]
@atruskot Hi! Thanks for the request.
We had this bug but fixed it in the following(`0.13.16`) version. Please, check it.

<img width="1483" alt="Screenshot 2022-06-03 at 09 34 36" src="https://user-images.githubusercontent.com/14113673/171800215-9f8e5122-6e98-4230-9273-decb1d6b4c23.png">

If it won't work for you please reopen the issue.

**Comment by atruskot** [comment]
Thanks, @AlexVarchuk. I've confirmed that this issue is resolved in `0.13.16`. Thanks for the quick response!
